evennia/web/webclient/static/webclient/js/plugins/options.js

```diff
@@ -137,7 +137,7 @@ let options_plugin = (function () {
         // is helppopup set? and if so, does this Text have type 'help'?
         if ('helppopup' in options && options['helppopup'] ) {
             if (kwargs && ('type' in kwargs) && (kwargs['type'] == 'help') ) {
-                $('#helpdialogcontent').append('<div>'+ args + '</div>');
+                $('#helpdialogcontent').prepend('<div>'+ args + '</div>');
```

Comment:
By changing the append to a prepend it causes it to push down the current "stack" of help information.  Makes for a better experience.